docs/content/developer/getting-started/graphql-rpc.mdx

```diff
@@ -363,10 +363,10 @@ mutation ($tx: String!, $sigs: [String!]!) {
 
 ## Other examples
 
-You can find other examples in the [repository](https://github.com/iotaledger/iota/tree/releases/iota-graphql-rpc-v2024.1.0-release/crates/iota-graphql-rpc/examples), grouped into sub-directories. For example, there are directories for [transaction block effects](https://github.com/iotaledger/iota/tree/releases/iota-graphql-rpc-v2024.1.0-release/crates/iota-graphql-rpc/examples/transaction_block_effects), [protocol configs](https://github.com/iotaledger/iota/tree/releases/iota-graphql-rpc-v2024.1.0-release/crates/iota-graphql-rpc/examples/protocol_configs), [stake connection](https://github.com/iotaledger/iota/tree/releases/iota-graphql-rpc-v2024.1.0-release/crates/iota-graphql-rpc/examples/stake_connection), and more.
+You can find other examples in the [repository](https://github.com/iotaledger/iota/tree/develop/crates/iota-graphql-rpc/examples), grouped into sub-directories. For example, there are directories for [transaction block effects](https://github.com/iotaledger/iota/tree/develop/crates/iota-graphql-rpc/examples/transaction_block_effects), [protocol configs](https://github.com/iotaledger/iota/tree/develop/crates/iota-graphql-rpc/examples/protocol_configs), [stake connection](https://github.com/iotaledger/iota/tree/develop/crates/iota-graphql-rpc/examples/stake_connection), and more.
 
 :::info
-Examples in the repository are designed to work with the version of GraphQL built at the same revision. The links above point to examples intended for [GraphQL v2024.1](https://github.com/iotaledger/iota/tree/releases/iota-graphql-rpc-v2024.1.0-release), the latest production version at the time of writing.
+Examples in the repository are designed to work with the version of GraphQL built at the same revision. The links above point to examples intended for [GraphQL v2024.1](https://github.com/iotaledger/iota/tree/develop/crates/iota-graphql-rpc-v2024.1.0-release), the latest production version at the time of writing.
 :::
 
```

Comment:
What is the plan with this section? Keep a single link to the examples? Remove it?

Reply:
Let's link to the rolling examples folder instead: https://github.com/iotaledger/iota/tree/develop/crates/iota-graphql-rpc/examples

So remove references to a specific release.